main.cpp

```diff
@@ -175,6 +168,18 @@ int Tree::checkSquirrelResidents()
     return numberOfSquirrels;
 }
 
+int Tree::squirrelPopulationGrowth(int initialPopulation, int numberMonths)
+{
+    numberOfSquirrels = initialPopulation;
+    for (int i = 0; i < numberMonths; i = i+3 ) 
+    {
+        numberOfSquirrels = numberOfSquirrels + numberOfSquirrels*2; // population doubles every three months
```

Comment:
This is just `numSquirrels * 3`
- [ ] revise

Reply:
ah ha! I simplified the population growth math